resources/js/Components/CharacterDetail/CharacterDetailBox.tsx

```diff
@@ -3,10 +3,12 @@ import DownloadIcon from '@mui/icons-material/Download'
 import IosShareIcon from '@mui/icons-material/IosShare'
 import {Box, Button, Grid, Stack, Typography} from '@mui/material'
 import {Link} from 'Components'
-import React from 'react'
+import React, {useContext} from 'react'
 import styled from 'styled-components'
 import route from 'ziggy-js'
 
+import {UserContext} from '../../Layouts/ApplicationLayout/UserContext'
```

Comment:
don't like this import path

Reply:
On a second glance, I don't like it either @willyyhuang 😆